### DISCUSSION TITLE
如何只构建dolphinscheduler-master单个docker镜像，而不用把所有docker镜像都构建一遍

### DISCUSSION THREAD

**Discussion by heartInGod** [apache/dolphinscheduler#14319] [Q&A]
每次构建docker镜像，都需要执行./mvnw 命令构建所有的docker镜像

**Comment by Slashoper** [comment]
cd  /bdp-dolphinscheduler/dolphinscheduler-master
docker build -t bdp-dolphin-master:3.1.4 -f ./src/main/docker/Dockerfile .

**Reply by Slashoper** [reply]
cd  /data/bdp-dolphinscheduler/dolphinscheduler-master
docker build -t bdp-dolphin-master:3.1.4 -f ./src/main/docker/Dockerfile .
